src/Components/Components/src/CascadingParameterState.cs

```diff
@@ -87,9 +87,9 @@ private static ReflectedCascadingParameterInfo[] GetReflectedCascadingParameterI
             return infos;
         }
 
-        private static ReflectedCascadingParameterInfo[] CreateReflectedCascadingParameterInfos(Type componentType)
+        private static ReflectedCascadingParameterInfo[]? CreateReflectedCascadingParameterInfos(Type componentType)
```

Comment:
Should we take this as an opportunity to improve some of our designs here? For example, in this case would it make sense to return an empty array and use that? Changes like that have the potential to simplify our code all over the place.

I would suggest we think about these things with a mentality of prefer avoiding null unless necessary.

Reply:
Typically, yes, sounds good! Individual cases may vary (don't know about this one).

Reply:
We could. This one's all internal to our code, so it's pretty cheap to update.